Solutions/CohesitySecurity/Playbooks/Alert_VM_Playbook/Alert_VM_Playbook.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Can remove this playbook. It was created for understanding and reference purpose only.

Reply:
this one is just removed.